dnf-docker-test/features/module-enable-1.feature

```diff
@@ -8,14 +8,48 @@ Feature: Enabling module stream
 
   Scenario: I can enable a module when specifying stream
        When I successfully run "dnf module enable ModuleA:f26"
-#TODO: add a check that the module is really enabled
+       Then a module ModuleA config file should contain
+          | Key     | Value |
+          | enabled | 1     |
+          | stream  | f26   |
+       And the command stdout should match regexp "'ModuleA:f26' is enabled"
 
   Scenario: I can enable a module when specifying both stream and corrent version
        When I successfully run "dnf module enable ModuleB:f26:1"
-#TODO: add a check that the module is really enabled
+       Then a module ModuleB config file should contain
+          | Key     | Value |
+          | enabled | 1     |
+          | stream  | f26   |
+          # version is ignored in the current libmodulemd (always -1)
+          #| version | 1     |
+       And the command stdout should match regexp "'ModuleB:f26:1' is enabled"
 
   Scenario: Enabling of an already enabled module would pass
        When I successfully run "dnf module enable ModuleA:f26"
+       Then a module ModuleA config file should contain
+          | Key     | Value |
+          | enabled | 1     |
+          | stream  | f26   |
+       And the command stdout should match regexp "'ModuleA:f26' is enabled"
 
   Scenario: I can enable a different stream of an already enabled module
        When I successfully run "dnf module enable ModuleA:f27 --assumeyes"
+       Then a module ModuleA config file should contain
+          | Key     | Value |
+          | enabled | 1     |
+          | stream  | f27   |
+       And the command stdout should match regexp "'ModuleA:f27' is enabled"
+
+  Scenario: Enabling a module doesn't install any packages
+        When I save rpmdb
+        And I successfully run "dnf -y module enable ModuleA:f26"
+        Then a module ModuleA config file should contain
+           | Key     | Value |
+           | enabled | 1     |
+           | stream  | f26   |
+        And rpmdb does not change
+
+  Scenario: module enable without specifying a name gives an error
```

Comment:
I'm going to move this to module-enable-3 which is about error handling. I'll send an updated version after any comments you might have on the rest of the PR.